libs/template/presets/storefront/experience/pages/registration-page.ts

```diff
@@ -0,0 +1,129 @@
+import {
+  CompositionLayout,
+  ExperienceComponent,
+} from '@spryker-oryx/experience';
+import { ButtonSize, ButtonType } from '@spryker-oryx/ui/button';
+import { Size, featureVersion } from '@spryker-oryx/utilities';
+
+export const registrationPage: ExperienceComponent | undefined =
+  featureVersion >= '1.1'
```

Comment:
What about have `registrationPage` without any versioning constraints, but only import it in `StaticExperienceFeature` with the version condition?